src/style.css

```diff
@@ -1 +1,85 @@
-/* add styles here */
+.search-bar {
+  display: flex;
+  margin-top: 20px;
+  margin-bottom: 20px;
```

Comment:
don't use both top and bottom margins to avoid margin collapse.
choose one and be consistent.
for example to add spacing between elements you can add for them only margin top